.github/workflows/upload-release-asset.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
```

Comment:
`workflow_dispatch` event might create ambiguous behaviours if the latest tag is different than the branch on which dispatch is triggered.
```suggestion
```